alg/crc32c.c

```diff
@@ -74,6 +79,60 @@ init(void)
 	assert(T0[0x80] == T_0_0x80);
 }
 
+#ifdef CPUSUPPORT_X86_CRC32
+/* Test whether CRC extensions and software code produce the same results. */
+static int
+crctest(const uint8_t buf[4])
```

Comment:
I'm not convinced that testing *4 bytes* is really helpful.  I mean, this test would pass even if `crc32q` isn't implemented.

How about taking the CRC of { 0x11, 0x22, 0x33, 0x44 ... 0xff }?